components/script/script_runtime.rs

```diff
@@ -761,22 +760,26 @@ unsafe extern "C" fn gc_slice_callback(
 ) {
     match progress {
         GCProgress::GC_CYCLE_BEGIN => GC_CYCLE_START.with(|start| {
-            start.set(Some(now()));
+            start.set(Some(SystemTime::now()));
             println!("GC cycle began");
         }),
         GCProgress::GC_SLICE_BEGIN => GC_SLICE_START.with(|start| {
-            start.set(Some(now()));
+            start.set(Some(SystemTime::now()));
             println!("GC slice began");
         }),
         GCProgress::GC_SLICE_END => GC_SLICE_START.with(|start| {
-            let dur = now() - start.get().unwrap();
+            let dur = SystemTime::now()
+                .duration_since(start.get().unwrap())
+                .unwrap();
             start.set(None);
-            println!("GC slice ended: duration={}", dur);
+            println!("GC slice ended: duration={:?}", dur);
         }),
         GCProgress::GC_CYCLE_END => GC_CYCLE_START.with(|start| {
-            let dur = now() - start.get().unwrap();
+            let dur = SystemTime::now()
+                .duration_since(start.get().unwrap())
+                .unwrap();
```

Comment:
The same comment here.

Reply:
See [previous conversation](https://github.com/servo/servo/pull/30639#discussion_r1375412348).